### ISSUE TITLE
Staking bitcoin as a security messure for the did creation

### ISSUE THREAD

**Issue by toescudero** [decentralized-identity/ion#269]
is there any implemented a mechanism like this? or can i be reffer to some spec or doc?

**Comment by csuwildcat** [comment]
Yes, this is implemented in ION: https://github.com/decentralized-identity/ion/blob/master/docs/design.md#value-locking
